package.json

```diff
@@ -335,6 +335,7 @@
     "redux": "4.1.0",
     "sass-loader": "10.2.1",
     "sinon": "7.5.0",
+    "skiafy": "github:fallaciousreasoning/skiafy",
```

Comment:
**Note:** I forked the package because it was expecting super out of date versions of `SVGO` and `JSDOM`. As we already run the SVGs through `SVGO` in the Leo repository, I just ripped them out, as it's a bit redundant running it twice.

In addition, I fixed a bug around `MoveTo` in SVGs. If this PR looks good, we can fork my repository into https://github.com/brave

@brave/sec-team

Reply:
Is the idea here that we'd be doing this dynamically with untrusted SVGs or are we planning to only run this conversion tool over trusted SVGs of ours?

Reply:
My plan here was to just run it over our trusted ones from the Leo repository. I've been talking to @sangwoo108 about it and was thinking it might be better to do this as part of the build step in Leo instead. Wdyt?

Reply:
Yeah, that seems like a much better plan to keep the dependency graph in core smaller.